### DISCUSSION TITLE
NutUI 团队组件库待办-期待您的声音

### DISCUSSION THREAD

**Discussion by Ymm0008** [jdf2e/nutui#1649] [Announcements]
当下，我们亟待解决的问题有：

1. picker 组件在小程序环境，数据量大的情况下，会出现卡顿的现象

2. Popup/OverLay 等弹层，需要解决滚动穿透的问题

3. 我们计划新增一个组件：拖拽排序组件

**Comment by wlchn** [comment]
@Ymm0008 
上面待办1：小程序picker性能优化：https://github.com/jdf2e/nutui/pull/1690

**Reply by Ymm0008** [reply]
感谢 PR，您的提交已处理。
